### DISCUSSION TITLE
Does WikiTraccs check if the Page exists in SharePoint?

### DISCUSSION THREAD

**Discussion by jlriverayuso** [WikiTransformationProject/wikitraccs-releases#134] [Q&A] [answered]
Hi All,

After running a lengthy migration for a few days, it seems my GUI closed when i was trying to move it around and then the console stated it was waiting for all retrieval threads to come to an end. 
due to the sheer amount of pages I was migrating is there a way that if re-migrate the same space, the app would review if the page exists and skip it?

thank you in advanced for the help.

**Comment by heinrich-ulbricht** [comment, marked as answer]
Hi @jlriverayuso, WikiTraccs does not overwrite previously migrated pages (unless explicitly told to do so). On every start of a migration it checks what's already there and skips it.

Also, the blue GUI can be opened and closed when the Console is running the migration. When having closed the GUI by accident, opening it should reconnect with the console as if nothing happened.

After interrupting a migration, just start the migration again and all should be fine.

**Comment by jlriverayuso** [comment]
@heinrich-ulbricht is there an easier way to tell WikiTraccs to update the SharePoint Page it found as being out of date? i have been trying to update the pages following the instructions: [https://www.wikitransformationproject.com/docs/reference/update-old-pages/](https://www.wikitransformationproject.com/docs/reference/update-old-pages/ )but still get they are out of date when i run the migration again.

**Reply by heinrich-ulbricht** [reply]
@jlriverayuso Is WikiTraccs not updating the pages in SharePoint, or is it updating the pages, but still reporting them as being out of date?

Could you please verify that the pages are indeed updated?

I'm trying to figure out if the page update or the reporting of the update fails.

**Reply by jlriverayuso** [reply]
i believe its both. I ran it again and the reporting still shows is out of date. the page was also not updated.

**Reply by heinrich-ulbricht** [reply]
@jlriverayuso I see. Pages should be updated and, by the way, you found the correct documentation (thank you for that)!

Could you run another delta upgrade migration (where you'd expect pages to be updated) and send the following info to support@wikitransformationproject.com:
* old progress log file(s) with the "x" marker from the **input** folder (where they'd been put by you according to the [documentation](https://www.wikitransformationproject.com/docs/reference/update-old-pages/#how-to-tell-wikitraccs-to-update-outdated-pages))
* [common log files](https://www.wikitransformationproject.com/docs/troubleshooting/troubleshooting-strategies/#common-log-files) for the migration run
* [progress log files](https://www.wikitransformationproject.com/docs/reference/monitoring-confluence-to-sharepoint-migration-progress/#using-progress-log-files-to-get-insights) for the migration run (newly created, mind the date and time in the file name)
* 3 page IDs of pages you'd expect to be updated - I'll use those to filter the logs

I should quickly be able to find out what's going on. Thank you.

**Reply by jlriverayuso** [reply]
@heinrich-ulbricht sent it your way, apologies on the delay.
